test/embulk/input/google_analytics/test_plugin.rb

```diff
@@ -379,13 +379,37 @@ def time_series
               @config = embulk_config(conf)
             end
 
-            sub_test_case "no records fetched" do
+            sub_test_case "no records fetched and last_record_time is nil" do
```

Comment:
Should use fixture syntax to merge these `sub_test_case`

Reply:
Done